crates/criticaltrust/src/signatures/keychain.rs

```diff
@@ -20,6 +24,10 @@ impl Keychain {
     pub fn new(trust_root: &PublicKey) -> Result<Self, Error> {
         let mut keychain = Self {
             keys: HashMap::new(),
+            revocation_info: RevocationInfo::new(
+                Vec::<Vec<u8>>::new(),
+                datetime!(2025-01-01 0:00 UTC),
```

Comment:
This isn't test code, having static near-future dates in code like this is likely to bite us in the behind later.

Looking at the code, does `new` and `load_all` both exist? Is a `new()` constructed keychain actually usable? Maybe `load_all` should be the constructor.

Reply:
I left the `load_all` as is. I am unsure why there was a test that kept failing for me and key verification failed. I think quite some part of it is dependent on the creation of the Keychain first.